mlflow/ml-package-versions.yml

```diff
@@ -830,6 +830,20 @@ autogen:
         ]
     run: pytest tests/autogen
 
+gemini:
+  package_info:
+    pip_release: "google-generativeai"
+    module_name: "google.generativeai"
+    install_dev: |
+      pip install git+https://github.com/google-gemini/generative-ai-python
+  autologging:
+    minimum: "0.8.0"
+    maximum: "0.8.3"
+    python:
+      "== dev": "3.10"
```

Comment:
nit: This means we will run the test with Python 3.10 *only for the "dev" version*. Do we need this for Gemini package?

Reply:
Thanks, let me take it out!